Resident-Teams/README.md

```diff
@@ -1,3 +1,9 @@
+---
+layout: default
+title: Resident Teams
```

Comment:
I wonder if a better title is "Products of the Lab"

Technically, teams are products, but then we also point to products :)